### PR TITLE
Chat fixes

### PR DESCRIPTION
**Pull request by tarasbashuk**

### CODE REVIEW

File: client/src/components/Chat/Chat.jsx [hunk 1]
```diff
@@ -10,14 +10,27 @@ import { getAllChats, getMessagesForChat } from '../../actions/chat'
...
+  ownMessageSend,
+  chatsLoading,
+  isLastPageInChat
+}) => {
   const classes = useStyles()
   const selectedChatId = +useParams().chatId
+  const loadContentHandler = getMessagesForChat.bind(null, selectedChatId)
+
+  useEffect(() => {
+    getMessagesForChat(selectedChatId, 0, 7, true)
```
Comment (RostyslavStepanchuk) [inline client/src/components/Chat/Chat.jsx]: If you don't mind, please replace literals with constants that explain meaning of literals

File: client/src/components/Chat/ChatDetails/ChatDetails.jsx [hunk 2]
```diff
@@ -9,7 +9,16 @@ import SendMessage from './SendMessage/SendMessage'
...
 
-const ChatDetails = ({ authUser, chat, messages, className }) => {
+const ChatDetails = ({
+  authUser,
+  chat,
+  messages,
+  className,
+  messagesLoading,
+  loadContentHandler,
+  ownMessageSend,
```
Comment (RostyslavStepanchuk) [inline client/src/components/Chat/ChatDetails/ChatDetails.jsx]: I you were frustrated with "isAllValidationsDone", "ownMessageSend" should also annoy you ;)

File: client/src/components/Chat/ChatDetails/ChatMessages/ChatMessages.jsx [hunk 3]
```diff
@@ -1,35 +1,86 @@
...
+    <InfiniteScroll
+      isReverseDirection
+      contentIsLoading={messagesLoading}
+      contentArr={messages}
+      loadContentHandler={loadContentHandler}
+      size={7}
+      throttleDelay={1000}
+      isLastPage={isLastPageInChat}
+      scrollContainerStyles={{
+        height: '80vh',
```
Comment (RostyslavStepanchuk) [inline client/src/components/Chat/ChatDetails/ChatMessages/ChatMessages.jsx:45]: On homepage posts container is cut. Check please, maybe problem is with this property
image

**Review by RostyslavStepanchuk — APPROVED** [review]

File: client/src/components/Chat/Chat.jsx (hunk 1 above)
Comment (tarasbashuk) [reply client/src/components/Chat/Chat.jsx]: Sure

File: client/src/components/Chat/ChatDetails/ChatDetails.jsx (hunk 2 above)
Comment (tarasbashuk) [reply client/src/components/Chat/ChatDetails/ChatDetails.jsx]: ))

File: client/src/components/Chat/ChatDetails/ChatMessages/ChatMessages.jsx (hunk 3 above)
Comment (tarasbashuk) [reply client/src/components/Chat/ChatDetails/ChatMessages/ChatMessages.jsx:45]: Thanks, forgot to remove  height: '80vh' from scrollContainerStyles

**Pull request merged**